api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -100,6 +100,7 @@ type NodePoolSpec struct {
 
 	// Management specifies behavior for managing nodes in the pool, such as
 	// upgrade strategies and auto-repair behaviors.
+	// +kubebuilder:default:={autoRepair: false, upgradeType: "Replace", replace: {strategy: "RollingUpdate", rollingUpdate: {maxSurge: 1, maxUnavailable: 0 }}}
```

Comment:
this is still defaulting to replace strategy, let's drop it

Reply:
oh no, good catch!

Reply:
fixed. I added this defaulting to the kubevirt platform in our optional defaulting webhook, but removed it from the API's defaulting